src/serverenvironment.cpp

```diff
@@ -1194,14 +1191,16 @@ void ServerEnvironment::step(float dtime)
 		}
 	}
 
+	// placeholder for the "real" time passed
+	float elapsed_time;
+
 	/*
 		Mess around in active blocks
 	*/
-	if (m_active_blocks_nodemetadata_interval.step(dtime, m_cache_nodetimer_interval)) {
+	if (m_active_blocks_nodemetadata_interval.step(dtime, m_cache_nodetimer_interval,
+		&elapsed_time)) {
```

Comment:
http://dev.minetest.net/Code_style_guidelines#Spaces
> When breaking conditionals indent following lines of the conditional with two tabs and the statement body with one tab. [...]

i.e. Needs an indent (tab) more for conditions that are continued on multiple lines.